### PR TITLE
send the email via the background worker in the payment request

### LINKED ISSUE #10340: Email should be sent via the background worker in the payment request

### PR DESCRIPTION
**Pull request by manassolanki**
Fixes https://github.com/frappe/erpnext/issues/10340

### CODE REVIEW

File: erpnext/accounts/doctype/payment_request/payment_request.py [hunk 1]
```diff
@@ -143,9 +144,10 @@ def create_payment_entry(self, submit=True):
 
 	def send_email(self):
 		"""send email with payment link"""
-		frappe.sendmail(recipients=self.email_to, sender=None, subject=self.subject,
-			message=self.get_message(), attachments=[frappe.attach_print(self.reference_doctype,
-			self.reference_name, file_name=self.reference_name, print_format=self.print_format)])
+		email_args = {"recipients":self.email_to, "sender":None, "subject":self.subject,
+			"message":self.get_message(), "attachments":[frappe.attach_print(self.reference_doctype,
+			self.reference_name, file_name=self.reference_name, print_format=self.print_format)]}
+		enqueue(method=frappe.sendmail, queue='short', timeout=300, async=True, now=True, **email_args)
```
Comment (mbauskar) [inline erpnext/accounts/doctype/payment_request/payment_request.py]: @manassolanki,

`now=True` will execute the sendmail method immediately, remove the now args.

**Review by mbauskar — CHANGES_REQUESTED** [review]

**Comment by mbauskar** [comment]
@manassolanki please check my comments, closing the PR for now, Please reopen the PR once fixed

File: erpnext/accounts/doctype/payment_request/payment_request.py (hunk 1 above)
Comment (manassolanki) [inline erpnext/accounts/doctype/payment_request/payment_request.py]: Yes, I have given `now=True` purposefully.

**Comment by manassolanki** [comment]
@mbauskar Done

**Pull request merged**